packages/core/review-workflows/shared/contracts/review-workflows.ts

```diff
@@ -133,7 +136,11 @@ namespace Update {
 namespace Create {
   export interface Request {
     body: {
-      data: Omit<Workflow, 'id' | 'createdAt' | 'updatedAt'>;
+      data: Omit<Workflow, 'id' | 'createdAt' | 'updatedAt'> & {
+        // We cannot use id because the stage could be not created yet
+        // And considering name is unique inside the workflow, name is a better option
+        stageRequiredForPublish?: Stage['name'] | null;
```

Comment:
It wouldn't change anything here because it's a string too, but I would find it more explicit if you added an empty string in this union: it would explain what you use to store the "any" value, since it's not exactly a stage name

Reply:
The thing is that I don't want users to send empty string when "any" value is selected. I want them to send null and on frontend I catch that case to replace "" for null (only because SingleSelect component doesn't accept null as value of one option)